definitions/variable/tag_electricity.yaml

```
@@ -17,6 +17,12 @@
         description: Coal gases
     - Gas|Hydrogen:
         description: Hydrogen
+    - Gas|Synthetic Gas:
+        description: Synthetically produced gas
```

For consistency with the following two variables
```suggestion
        description: Synthetically produced methane
```